public/manifest.json

```diff
@@ -1,14 +1,33 @@
 {
-    "short_name": "OBS-web",
-    "name": "OBS-web - The easiest way to control OBS remotely",
-    "icons": [
+  "theme_color": "#00d1b2",
+  "background_color": "#00d1b2",
+  "display": "standalone",
+  "scope": "/",
+  "start_url": "/",
+  "short_name": "OBS-web",
+  "app_name": "OBS-web",
+  "description": "OBS-web - The easiest way to control OBS remotely",
+  "prefer_related_applications": false,
+  "icons": [
       {
-        "src": "favicon.png",
-        "sizes": "96x96",
-        "type": "image/png"
+          "src": "/icon/icon-192x192.png",
+          "sizes": "192x192",
+          "type": "image/png"
+      },
+      {
+          "src": "/icon/icon-256x256.png",
+          "sizes": "256x256",
+          "type": "image/png"
+      },
+      {
+          "src": "/icon/icon-384x384.png",
+          "sizes": "384x384",
+          "type": "image/png"
+      },
+      {
+          "src": "/icon/icon-512x512.png",
+          "sizes": "512x512",
+          "type": "image/png"
       }
-    ],
-    "start_url": "/index.html",
-    "display": "standalone",
-    "orientation": "landscape"
```

Comment:
With orientation, there is also a question.
Maybe it's better not to specify it so that the user chooses the right one?
I use only the `portrait` on my phone.

Reply:
In my opinion it's good to keep default landscape orientation, especially on tablet it's much more convenient.